Client/Application/Migration.swift

```diff
@@ -23,6 +25,9 @@ class Migration {
             documentsDirectoryCleanup()
             Preferences.Migration.documentsDirectoryCleanupCompleted.value = true
         }
+        
+        //`.migrate` is called in `BrowserViewController.viewDidLoad()`
+        braveCoreBookmarksMigrator = BraveCoreMigrator()
```

Comment:
do we have to initialize it if the sync migration has happened already?

Reply:
There are some observers that might use it to check if enabled. 
For now we don't have to allocate it. I fixed it :)